app/src/main/java/org/simple/clinic/home/patients/PatientScreenRepository.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.home.patients
```

Comment:
Reword the commit message to be a bit more descriptive? I can suggest "Display illustration in the home screen".